commons-mfa/src/main/java/org/exoplatform/mfa/api/MfaService.java

```diff
@@ -99,6 +109,11 @@ public boolean addRevocationRequest(String username, String mfaType) {
       revocationRequest.setUser(username);
       revocationRequest.setType(mfaType);
       mfaStorage.createRevocationRequest(revocationRequest);
+      try {
+        listenerService.broadcast(new Event("mfa.listener.create.revocation.request", null, username));
+      } catch (Exception e) {
+        LOG.error("Error when broadcasting dlp detect positive item", e);
```

Comment:
Please fix Log message, not DLP detected  positive item

Reply:
done